### ISSUE TITLE
NPE is logged while playing some files

### ISSUE THREAD

**Issue by helenayele** [debrief/debrief#4373]
When I play shapes.rep file , I get the following NPE logged. 
[NPE while playing.txt](https://github.com/debrief/debrief/files/3381806/NPE.while.playing.txt)
But I get no issue when I play boat1.rep and boat2.rep or any other files

### Steps:

1. Just open shapes.rep file
2. Open Time menu and Select Play

**Comment by IanMayo** [comment]
I can't reproduce this.  

You're on `develop`, right?

Did you open other files before `shapes.rep` - or was it the first file you opened?

**Comment by helenayele** [comment]
Yes, I was in develop, and opened other files as well before opening shapes.rep file.

**Comment by helenayele** [comment]
### Steps to reproduce:
1. Launch D-Lite
2. Open shapes.rep
3. Go to Time menu and start playing
4. Change to Snail mode and start playing
This time the exception is logged.

**Comment by IanMayo** [comment]
@helenayele - I still can't reproduce. Would you mind making a video, please?

**Comment by helenayele** [comment]
Sure, here is the video:
https://screencast-o-matic.com/watch/cqiX2QODc3

**Comment by helenayele** [comment]
@IanMayo , I am getting the below logs while playing the shape.rep file:
2019-07-12T12:16:35.080-0700  WARNING  Possible use of "Mercator_1SP" projection outside its valid area.
Longitude 2356361°59.5'W is out of range (±180°).
Latitude 2434430°43.6'N is out of range (±90°).
Error:Failure in projection transform in toScreen operation:Latitude 2434430°43.6'N is too close to a pole. from location: 21°24'00.00"N 020°37'03.64"W 
Error:Failure in projection transform in toScreen operation:Latitude 2459998°12.0'N is too close to a pole. from location: 21°29'12.07"N 020°41'22.57"W 
no coords for: 21°29'12.07"N 020°41'22.57"W 
Error:Failure in projection transform in toScreen operation:Latitude 2425651°08.2'N is too close to a pole. from location: 21°24'59.25"N 020°36'00.00"W 
no coords for: 21°24'59.25"N 020°36'00.00"W 
Error:Failure in projection transform in toScreen operation:Latitude 2447484°51.4'N is too close to a pole. from location: 21°36'00.00"N 021°32'39.31"W 
Error:Failure in projection transform in toScreen operation:Latitude 2401891°11.6'N is too close to a pole. from location: 21°24'59.25"N 020°36'00.00"W 
no coords for: 21°24'59.25"N 020°36'00.00"W 
Error:Failure in projection transform in toScreen operation:Latitude 2324130°54.7'N is too close to a pole. from location: 20°33'36.00"N 019°31'12.00"W 
Error:Failure in projection transform in toScreen operation:Latitude 2312315°25.0'N is too close to a pole. from location: 21°42'00.00"N 020°55'03.77"W 
Error:Failure in projection transform in toScreen operation:Latitude 2425651°08.2'N is too close to a pole. from location: 21°36'59.25"N 020°48'00.00"W 
no coords for: 21°36'59.25"N 020°48'00.00"W 
Error:Failure in projection transform in toScreen operation:Latitude 2507106°41.6'N is too close to a pole. from location: 22°06'00.00"N 021°30'00.00"W 
Error:Failure in projection transform in toScreen operation:Latitude 2418940°38.2'N is too close to a pole. from location: 21°23'12.07"N 020°35'22.60"W 
no coords for: 21°23'12.07"N 020°35'22.60"W 
Error:Failure in projection transform in toScreen operation:Latitude 2312315°25.0'N is too close to a pole. from location: 21°18'18.31"N 020°31'00.48"W 
no coords for: 21°18'18.31"N 020°31'00.48"W 
Error:Failure in projection transform in toScreen operation:Latitude 2399937°27.3'N is too close to a pole. from location: 21°17'12.07"N 020°30'37.38"W 
no coords for: 21°17'12.07"N 020°30'37.38"W 

### Video:
https://screencast-o-matic.com/watch/cqilQ3Ob31
